packages/@ember/-internals/runtime/lib/mixins/action_handler.d.ts

```diff
@@ -1,3 +1,7 @@
 import { Mixin } from '@ember/-internals/metal';
 
-export default class ActionHandler extends Mixin {}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface ActionHandler {}
+declare const ActionHandler: Mixin;
```

Comment:
Will presumably need a rebase now that #19948 has also updated this?

Reply:
Yes